rpc/src/types/errors.rs

```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+use serde_json::{json, Value};
+use std::collections::HashMap;
+
+#[derive(Serialize, Deserialize)]
+pub struct ResponseError {
+  title: String,
+  description: String,
+  r#type: String,
+  properties: HashMap<String, Value>,
+  required: Vec<String>,
+  #[serde(rename(
+    serialize = "additionalProperties",
+    deserialize = "additionalProperties"
+  ))]
+  additional_properties: bool,
+}
+
+impl ResponseError {
+  pub fn get_string(&self) -> String {
+    json!(self).to_string()
+  }
```

Comment:
If this is to be an equivalent of a `to_string()` method, this is breaking convention.

[the `to_`, `as_` and `into_` methods have a strong meaning in rust contexts](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv), so seeing a `get_string()` call might startle more than a few devs (including me hahaha).